docs/layers/gpu-grid-layer.md

```diff
@@ -6,13 +6,13 @@
   <img src="https://img.shields.io/badge/lighting-yes-blue.svg?style=flat-square" alt="lighting" />
 </p>
 
-# GPUGridLayer
+# GPUGridLayer (WebGL2)
 
 The `GPUGridLayer` renders a grid heatmap based on an array of points.
```

Comment:
I wonder if we should add a sentence to the top of GPUGridLayer/CPUGridLayer docs that directs non-advanced users to use the GridLayer. It can be quite confusing since all three classes are presented as peers in the documentation.

Reply:
You may also consider adding this tag: https://github.com/uber/deck.gl/blob/master/website/contents/pages.js#L357

Reply:
Done.